ConvertFrom-SID.ps1

```diff
@@ -123,10 +148,10 @@ function ConvertFrom-SID {
 
             #Display the results
             New-Object -TypeName PSObject -Property @{
-                SID = $id
+                SID = $id;
```

Comment:
Hiyo!  Don't need the semi-colon here.  Going to test this _suggestion_ thingy!

```suggestion
                SID = $id
```

Reply:
oops - sorry, was playing. i missed that before submitting